pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -123,7 +123,7 @@ type containerManagerImpl struct {
 	capacity v1.ResourceList
 	// Absolute cgroupfs path to a cgroup that Kubelet needs to place all pods under.
 	// This path include a top level container for enforcing Node Allocatable.
-	cgroupRoot string
+	cgroupRoot CgroupName
```

Comment:
is this comment still accurate?  its still the internal canonical name, no?

this pr is a nice a cleanup, but the usage of string meant absolute name, and CgroupName had meant internal canonical name.

Reply:
Yes, I think the comment is still accurate, since CgroupName is still an absolute path (even though it's encoded as a slice of strings rather than a single string.) If you'd like me to make that more explicit, I can do that, but I think it's fine for now...